posts/2022-06-07-microprofile-graphql-2-22006.adoc

```diff
@@ -77,6 +78,123 @@ To enable the MicroProfile GraphQL 2.0 feature, add it to your `server.xml`:
 </featureManager>
 ----
 
+[#db2]
+===  Improved DB2 Query Performance
+This new JPA enhancement improves EclipseLink's DB2 and DB2 z/OS support by enabling parameter binding for JPA specification defined functions.
+
+For DB2 and DB2 for z/OS, EclipseLink's JPQL parameter binding is disabled when function expressions (SQRT, ABS, SUBSTR, COALESCE, etc.) exist within a JPQL query. This feature will allow EclipseLink to bind JPQL parameters for function expressions that accept untyped parameter markers on DB2 and DB2 z/OS. 
+This feature will also allow EclipseLink to enable/disable parameter binding per expression, within a query, rather then for the entirety of the query.
```

Comment:
```suggestion
This update also allows EclipseLink to enable and disable parameter binding per expression within a query, rather than for the entirety of the query.
```